pkg/controller/cache/managed.go

```diff
@@ -168,7 +179,8 @@ func (e *external) Update(ctx context.Context, mg resource.Managed) (managed.Ext
 	instance := &redis.Instance{}
 	fqn := cloudmemorystore.GetFullyQualifiedName(e.projectID, i.Spec.ForProvider, meta.GetExternalName(i))
 	cloudmemorystore.GenerateRedisInstance(fqn, i.Spec.ForProvider, instance)
-	_, err := e.cms.Projects.Locations.Instances.Patch(fqn, instance).Context(ctx).Do()
+	updateMask := strings.Join([]string{"display_name", "labels", "memory_size_gb", "redis_configs"}, ",")
```

Comment:
Just noting for other folks that this is being added back in because it was mistakenly removed, but it is not specifically related to this PR.